### PR TITLE
created suggested topics content

### PR DESCRIPTION
**Pull request by anaxceron**
Wrote suggestions for future GitHub demo topics

### CODE REVIEW

**Comment by anaxceron** [comment]
@DS1NMOD -- Per our homework instructions, please provide a comment in this PR using the comment feature. The content of the comment doesn't matter, but you do get "bonus points" if you leave an inline comment.

**Comment by IgorCATech** [comment]
Great and prompt execution of the assignment, Anna!
I'll let you include an inline comment if you want and then will merge the pull request. 

As for your suggestions, they make a lot of sense and I'll be happy to talk about those features in future workshops if we manage to get approval from our management and gather enough participants :) If not, I can sync up with you individually to provide an individual session.

File: lists exercise.md [hunk 1]
```diff
@@ -1,21 +1,7 @@
-# Place an order on our online shop
+# Suggestions for the next GitHub demo
 
-To place an order on our online shop, follow these steps:
+It would be good to build off the basic understanding folks now have of working in GitHub. Some possible future demos could include using VS Code and GitHub Desktop, as well as any extensions for these that tech writers would find useful. I think it'd also be good to include an overview of DCOs. 
 
-1. Find the products you want to purchase
+Also, personally, I'd like to learn how to use git commands to commit and/or create pull requests as a way to kick my knowledge up a notch or two.
```
Comment (anaxceron) [inline lists exercise.md:6]: This is my inline comment!

**Comment by anaxceron** [comment]
> 
I've added an inline comment. I'll keep an eye out for future demos -- thanks!

File: lists exercise.md (hunk 1 above)
Comment (IgorCATech) [reply lists exercise.md:6]: Well done! More points to you, Anna :star:

**Comment by DS1NMOD** [comment]
This is my inline comment - thanks Ana

**Pull request merged**